README.md

```diff
@@ -1,142 +1,232 @@
+  immediate response to continue processing. WSO2 EI acts as a Message Broker server.  
+
+**Business-process profile**: If your integration story contains human interactions (i.e., approval process) WSO2 EI  
+  provides BPS/BPMN and human task capabilities to develop work-flows.
+
+**Analytics profile**: Provides time-based/count-based analytics, stats, and monetization capability at different points of 
+  your integration flow implemented by combining the above profiles.    
+  
+All above aspects are seamlessly cross-supportive and, together, makes WSO2 EI a complete powerful middleware solution that
+helps you to digitalize your business with more agility.
+
+WSO2 EI has a **cloud native offering** as well named **[WSO2 Micro Integrator](https://github.com/wso2/micro-integrator)** that
+enables developers implement composite microservices and integrate them. It contains a microservices friendly 
+implementation of Enterprise Service Bus profile.
+
+[![WSO2 Enterprise Integrator](gh-docs/images/wso2-ei-overview.png?raw=true "WSO2 Enterprise Integrator")](https://wso2.com/integration/features/)
+
+
+## Features
+
+
+- **Support for all EIP Patterns**: Integrate applications following 
+  [standard enterprise integration patterns](https://en.wikipedia.org/wiki/Enterprise_Integration_Patterns) 
+- **Faster message mediation**: [Passthrough HTTP Transport](https://wso2.com/library/articles/2013/12/demystifying-wso2-esb-pass-through-transport-part-i/) 
+  for faster message mediation through WSO2 EI
+- **Supports multiple transports/messaging standards**: Interconnect applications that support different 
+  protocols (i.e., HTTP and JMS)
+- **Supports numerous formats and protocols**: Interconnect applications that work with different message 
+  formats (i.e., XML and Json)
+- **As a gateway** : Front operations within the operations as a managed, secured proxy services/APIs.  
+- **Offer QOS to services/APis**: Supports throttling, caching responses for faster mediation, circuit breaking, 
+  applying security 
+- **Secure enterprise**: Apply Oauth 2.0, Saml SSO, Keberos for services/APIs. WS-security support for proxy services 
+- **Service orchestration**: Enable to interconnect a set of APIs and web services and expose them as a single API or 
+  a web service 
+- **Database integration**: Enable to expose data as services and APIs, stream data and listen for data changes and 
+  trigger events 
+- **Handle many concurrent HTTP(S) connections**: Ability to serve multiple HTTP connections concurrently using 
+  reactor pattern and Java NIO
+- **Event publishing, logging and auditing**: Server logs, audit logs, trace logs with information of executions within 
+  server to different levels and ability to integrate with popular log analytic software 
+- **170+ connectors**: Connecting Web APIs / Cloud services with on-premise enterprise applications
```

Comment:
we have 186 connector, shall we state 185+

Reply:
ACK. Updated.